pkg/persistence/config/writer/config_writer_test.go

```diff
@@ -1087,6 +1087,67 @@ func TestWriteConfigs(t *testing.T) {
 			},
 			expectedErrs: []string{"config.Segment"},
 		},
+		{
```

Comment:
Is it possible to put this in a separate test? Maybe even in a separate file?
This table taste is huge.

Reply:
We discussed and the proper refactoring is going to be as a separate task